tembo-operator/src/extensions.rs

```diff
@@ -17,6 +17,30 @@ lazy_static! {
     static ref VALID_INPUT: Regex = Regex::new(r"^[a-zA-Z]([a-zA-Z0-9]*[-_]?)*[a-zA-Z0-9]+$").unwrap();
 }
 
+
+// mapping of extension names to their trunk project names
+lazy_static! {
+    pub static ref TRUNK_MAP: HashMap<&'static str, &'static str> = {
+        let mut m = HashMap::new();
+        m.insert("vector", "pgvector");
+        m.insert("embedding", "pg_embedding");
+        m.insert("pgml", "postgresml");
+        m.insert("columnar", "hydra_columnar");
+        m.insert("pg_currency", "pgcurrency");
```

Comment:
I think this one is just `currency`
```suggestion
        m.insert("pg_currency", "currency");
```

Reply:
Sorry, other way around :upside_down_face:

Reply:
https://pgt.dev/extensions/pg_currency

which extension is this one, @ianstanton ?

Reply:
typo though, should be `_` right?

Reply:
@ianstanton , go ahead and add any other extension you know have a name mismatch in other PRs.

Reply:
Looks good, that one is `currency`